lms/djangoapps/courseware/module_render.py

```diff
@@ -518,6 +519,9 @@ def rebind_noauth_module_to_user(module, real_user):
     # to the Fragment content coming out of the xblocks that are about to be rendered.
     block_wrappers = []
 
+    if settings.FEATURES.get("LICENSING", False):
+        block_wrappers.append(wrap_with_license)
```

Comment:
@singingwolfboy you could add it to the `PreviewModuleSystem` here (https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/preview.py#L157-L171) to enable it in Studio preview.